docs/guides/tracks.md

```diff
@@ -22,10 +22,10 @@ Once you have your WebVTT file created, you can add it to Video.js using the tra
 ```html
 <video id="example_video_1" class="video-js vjs-default-skin"  
   controls preload="auto" width="640" height="264"  
-  data-setup='{"example_option":true}'>  
- <source src="http://video-js.zencoder.com/oceans-clip.mp4" type='video/mp4' />  
- <source src="http://video-js.zencoder.com/oceans-clip.webm" type='video/webm' />  
- <source src="http://video-js.zencoder.com/oceans-clip.ogv" type='video/ogg' />  
+  data-setup="{'example_option':true}">  
```

Comment:
given that the value given to `data-setup` is JSON, this cannot have single quotes around the key which is why single quotes were used for the whole value.
The changes to the `type` attribute below are good, though.